apps/meteor/tests/e2e/omnichannel/omnichannel-business-hours.spec.ts

```diff
@@ -0,0 +1,144 @@
+import { faker } from '@faker-js/faker';
+import type { Page } from '@playwright/test';
+
+import { IS_EE } from '../config/constants';
+import { Users } from '../fixtures/userStates';
+import { OmnichannelBusinessHours } from '../page-objects';
+import { createAgent } from '../utils/omnichannel/agents';
+import { createBusinessHour } from '../utils/omnichannel/businessHours';
+import { createDepartment } from '../utils/omnichannel/departments';
+import { test, expect } from '../utils/test';
+
+test.use({ storageState: Users.admin.state });
+
+test.describe('OC - Business Hours', () => {
+  test.skip(!IS_EE, 'OC - Manage Business Hours > Enterprise Edition Only');
+
+  let poOmnichannelBusinessHours: OmnichannelBusinessHours;
+
+	let department: Awaited<ReturnType<typeof createDepartment>>;
+	let agent: Awaited<ReturnType<typeof createAgent>>;
+
+  test.beforeAll(async ({ api }) => {
+		department = await createDepartment(api);
+    agent = await createAgent(api, 'user2');
+    await api.post('/settings/Livechat_enable_business_hours', { value: true }).then((res) => expect(res.status()).toBe(200));
+    await api.post('/settings/Livechat_business_hour_type', { value: 'Multiple' }).then((res) => expect(res.status()).toBe(200));
+	});
+
+	test.afterAll(async ({ api }) => {
+		await department.delete();
+		await agent.delete();
+    await api.post('/settings/Livechat_enable_business_hours', { value: false }).then((res) => expect(res.status()).toBe(200));
+    await api.post('/settings/Livechat_business_hour_type', { value: 'Single' }).then((res) => expect(res.status()).toBe(200));
+	});
+
+  test.beforeEach(async ({ page }: { page: Page }) => {
+		poOmnichannelBusinessHours = new OmnichannelBusinessHours(page);
+	});
+
+  test('OC - Manage Business Hours - Create Business Hours', async ({ page }) => {
+    const BHName = faker.string.uuid();
+
+    await page.goto('/omnichannel');
+		await poOmnichannelBusinessHours.sidenav.linkBusinessHours.click();
+
+    await test.step('expect correct form default state', async () => {
+			await poOmnichannelBusinessHours.btnCreateBusinessHour.click();
+			await expect(poOmnichannelBusinessHours.btnSave).not.toBeVisible();
+			await expect(poOmnichannelBusinessHours.btnCancel).not.toBeVisible();
```

Comment:
Do you need these expects? Imho clicking the back button and then expecting input search should be enough